src/Assembler.hpp

```diff
@@ -47,6 +48,7 @@ namespace shasta {
     class LocalAlignmentGraph;
     class LocalReadGraph;
     class SegmentGraph;
+    class Reads;
```

Comment:
This forward definition should not be necessary, if we are including `Reads.hpp`.